examples/zero_copy/Chatter.msg

```diff
@@ -0,0 +1,3 @@
+uint64 timestamp
+char[4000000] data # shared memory backend can only deal with fixed size messages
+uint64 data_length
```

Comment:
I would spice this up by adding `count` field such that the listener can very that there are no dropped msgs. WDYT?